tools/KeyfileIOUtils.cpp

```diff
@@ -39,13 +40,21 @@ void outputReplicaKeyfile(uint16_t numReplicas,
          << "c_val: " << config.cVal << "\n"
          << "replica_id: " << config.replicaId << "\n"
          << "read-only: " << config.isReadOnly << "\n\n"
-         << "# RSA non-threshold replica public keys\n"
-         << "rsa_public_keys:\n";
+         << "# Non-threshold replica public keys\n"
```

Comment:
Did you make sure to change the code which parses this file on both projects?
If so, you should either document that this is a breaking change, since old files will no longer be supported, or add a version field for the configuration, parse it on the receiving end and act accordingly (with the default being the old format).